modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java

```diff
@@ -227,7 +224,7 @@ protected PortletURL getPortletURL() {
 
 			if (_groupIds != null) {
 				portletURL.setParameter(
-					"groupIds", StringUtil.merge(_groupIds, StringPool.COMMA));
+					"groupIds", StringUtil.merge(getGroupIds(), StringPool.COMMA));
```

Comment:
Why are we doing this?

Reply:
When using _groupIds as value for this parameter, it only used the Control Panel group id which made it never use the values returned from getGroupIds, which are the ones we need to use. You could also check [this](https://github.com/liferay/liferay-portal/blob/4c3862d7b511114e8a6ad8768602f5c1cdb9ab89/modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetCategoriesSelectorTag.java#L422) for reference.